src/components/SpeakerBlock/SpeakerBlock.js

```diff
@@ -3,45 +3,52 @@ import './speaker-block.scss';
 import SocialIcons from '../SocialIcons/SocialIcons';
 import LazyLoad from 'react-lazyload';
 import { isChrome } from '../../utils/environment';
+import { getAnchorName } from '../../utils/common';
 
 export default class SpeakerBlock extends Component {
 
   static propTypes = {
     data: toBe.object,
   };
 
+  static defaultProps = {
+    data: {},
+  }
+
   render() {
-    const { data } = this.props;
+    const {
+      data: { first_name, last_name, image_src, social_icons, position, company, description }
+    } = this.props;
     const imageExtension = isChrome() ? 'webp' : 'jpg';
 
     return (
       <div className="speaker-block container container-fluid">
         <div className="speaker-block-cols">
           <a
             href=""
-            name={`${data.first_name}-${data.last_name}`}
+            name={getAnchorName([first_name, last_name])}
```

Comment:
why array? just `getAnchorName(first_name, last_name)`

Reply:
Because I wanted to make it reusable and extensible.

Reply:
```js
name={`${first_name}-${last_name}`.replace(/\s/g, '-')}
```

Reply:
thanks! the shortest and the best